src/libostree/libostree-experimental.sym

```diff
@@ -89,3 +89,8 @@ global:
   ostree_repo_finder_override_get_type;
   ostree_repo_finder_override_new;
 } LIBOSTREE_2017.12_EXPERIMENTAL;
+
+LIBOSTREE_2017.14_EXPERIMENTAL {
+global:
+  ostree_remote_get_type;
+} LIBOSTREE_2017.13_EXPERIMENTAL;
```

Comment:
I hope it's not problem that we start defining this symbol version in our 2017.13 build.

Reply:
I don’t think it will be, since the identifiers in the `.sym` file don’t relate to the autoconf package version at all.

Reply:
It does have to be listed in the Debian `.symbols` file though.

Reply:
⇒ https://github.com/endlessm/ostree/pull/97